pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -597,6 +596,39 @@ public static String splitPart(String input, String delimiter, int index) {
     }
   }
 
+  /**
+   * @param input the input String to be split into parts.
+   * @param delimiter the specified delimiter to split the input string.
+   * @param index the specified index for the splitted parts to be returned.
+   * @param max the max count of parts that the input string can be splitted into.
+   * @return splits string on the delimiter with the limit count and returns String at specified index from the split.
```

Comment:
Currently, the argument name is `max` which is inconsistent with the javadoc here that uses `limit`. The [existing method](https://github.com/apache/pinot/blob/master/pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java#L579) uses `limit` and we can continue using the same instead of `max` and leave with the existing comment.

Reply:
updated, thanks